README.md

```diff
@@ -0,0 +1,126 @@
+# nep141-token-convertor
+
+The purpose of this contract is to provide conversion service for nep141 tokens in whitelist.
+
+Contents:
+
+- [Terminology](#terminology)
+- [Function specification](#function-specification)
+  - [Whitelist management](#Whitelist-management)
+  - [Create conversion pool](#Create-conversion-pool)
+  - [Transfer token to contract](#Transfer-token-to-contract)
+  - [View functions](#View-functions)
+- [Contract interfaces](#Contract-interfaces)
+  - [Pool creator interfaces](#Pool-creator-interfaces)
+  - [Whitelist admin interfaces](#whitelist-admin-interfaces)
+  - [Types of msg field in ft_transfer_call](#Types-of-msg-field-in-ft_transfer_call)
+    - [AddLiquidity](#AddLiquidity)
+    - [Convert](#Convert)
+
+## Terminology
+
+- `nep141`: [A standard interface for fungible tokens in near network.](https://nomicon.io/Standards/FungibleToken/Core)
+- `conversion pool`: A conversion pool contain a pair of tokens and allows user convert a kind of token to another at a certain rate.
+- `pool creator`: People who create a conversion pool.
+- `rate`: Pool creator can set converting rate when create pool. Then user can convert tokens by this immutable rate in this pool.
```

Comment:
Maybe we need to clarify that the `rate` here is for conversion from token A to token B. And the rate of opposite direction conversion is calculated by this `rate`.

Reply:
Ok,I'll do it.

Reply:
Here is my modification: 
- `rate`: Pool creator can set converting rate when creating pool.Then user will convert token by this immutable rate in this pool no matter what direction of conversion is.